damnit/backend/extract_data.py

```diff
@@ -163,12 +163,26 @@ def get_dset_value(ds):
             # SQlite doesn't like np.float32; .item() converts to Python numbers
             return value.item() if (value.ndim == 0) else value
 
+    def get_attrs(ds):
+        d = {}
+        for name, value in ds.attrs.items():
+            if name in {"max_diff", "summary_method"}:
+                continue  # These are stored separately
+
+            if isinstance(value, np.ndarray):
+                value = value.tolist()
+            elif isinstance(value, np.generic):  # Scalar
+                value = value.item()
```

Comment:
I think it'd be nice to move this logic into `Cell`.

Reply:
That doesn't work for much the same reason - the boundary between the context process (containing `Cell`) & this code is an HDF5 file, and h5py will return Numpy values.

I do want to revisit this layer as well, but I didn't want to do it as part of this PR.